modal/runner/engines/vllm.py

```diff
@@ -142,6 +140,6 @@ async def generate(self, payload: CompletionPayload, params):
                 "Failed generation", extra={"model": self.engine_args.model}
             )
             if payload.stream:
-                yield create_sse_data(e)
+                yield create_sse_data(create_response_text(e))
```

Comment:
not sure i grok the error payload shapes here fully..

i would have expected the `create_error_response(` or `create_error_text(` methods to be used -- but cant tell when each is appropriate.  some simplification/removal & docstrings for these helpers might be warranted soon